### PR TITLE
feat: support organization stats endpoint [EP-2548]

### PR DESCRIPTION
**Pull request by iacoshoria**
Changes:
- add support for organization stats endpoint;
- read organization stats permission & authorization guard;
- update membership-service to upsert any new groups, roles, permissions;

### CODE REVIEW

File: src/routers/OrganizationRouter.ts [hunk 1]
```diff
@@ -42,6 +46,45 @@ const getAdminRouter = (basePath: string = '/', routePath: string = '/management
...
+      const data = {
+        id: group?._id,
+        name: group?.name,
+        slug: group?.name,
+        description: group?.description,
+        collections: await countByQuery(CollectionModel, queryOptions.filter),
+        locations: await countByQuery(LocationModel, queryOptions.filter),
+        layers: await countByQuery(LayerModel, queryOptions.filter),
+        widgets: await countByQuery(WidgetModel, queryOptions.filter),
+        dashboards: await countByQuery(DashboardModel, queryOptions.filter),
```
Comment (dan-qc) [inline src/routers/OrganizationRouter.ts:77]: Can we do these calls in parallel?

**Review by dan-qc — APPROVED** [review]

File: src/routers/OrganizationRouter.ts (hunk 1 above)
Comment (iacoshoria) [reply src/routers/OrganizationRouter.ts:77]: I see what you did there.

**Pull request merged**